account_invoice_search_by_reference/__openerp__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# © 2017 Eficent Business and IT Consulting Services S.L.
```

Comment:
Can you please change it to 'Copyright' as per new [Readme template](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/model_name.py#L2)